src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -18,7 +18,7 @@
           <h2>{{ messageStore.currentMessage?.subject }}</h2>
 
           <div v-for="(message, index) in messageStore.currentThread" :key="index" class="small mt-6">
-            <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''"></span>
+            <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'You Replied'"></span>
```

Comment:
Hey I see you modified this check : <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''"></span>
 
^^^ is this intentional?? because Initiated from could have these values:
 
 export type InitiatedFrom = "Investigation" | "PortalUser" | "Registry";
 
So if Message is "Initiated From" Investigation it will still show : "You Replied"... No?

Reply:
might have been a lintfix thing?

Reply:
Good catch, yes we had a typescript error with the old condition. I created a helper function for this text and remove the v-html while at it.